### DISCUSSION TITLE
How to stop it from showing ls output by default?

### DISCUSSION THREAD

**Discussion by adhamsalama** [marlonrichert/zsh-autocomplete#431] [Q&A] [answered]
Hello, I really like this plugin.

However, I have a minor issue, it shows the output of "ls" by default when I don't enter any commands.
![image](https://user-images.githubusercontent.com/40252995/162439764-ad644f3f-81af-416d-8de7-3bf1fea90148.png)

If I cd into a folder that has a lot of files, it spits out all files and fills the terminal, which is a bit annoying.

**Comment by marlonrichert** [comment, marked as answer]
Just set `min-input` to `1`:
https://github.com/marlonrichert/zsh-autocomplete/blob/d6a0ff574679280af52258ee6d2dcd3857283270/.zshrc#L14

**Reply by adhamsalama** [reply]
Thanks!
